attestation-evidence-broker/README.md

```diff
@@ -0,0 +1,51 @@
+# Attestation Evidence Broker (AEB)
+
+## Introduction
+
+Attestation Evidence Broker (AEB) is a host service which to query attestation evidence for guest attestation agent.
```

Comment:
s/a host service which to query attestation evidence for guest attestation agent/a host service to satisfy the requests from guest for getting attestation evidence/

Reply:
Add two sentences:
In theory, the attester is responsible for generating the attestation evidence. In practice, certain TEE, e.g, SEV(-ES), needs the help from untrusted host side to retrieve the attestation evidence.

Currently, AEB only supports SEV(-ES) runtime attestation. In future, it will support Intel TDX.